test/LibraryManager.IntegrationTest/Microsoft.Web.LibraryManager.IntegrationTest.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net461</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
```

Comment:
Glob these.

Better yet, put all of the `<ItemType Remove="">` globs into a single ItemGroup for the TestSolution files so that they are in one place.

Reply:
These items are no longer useful, so delete them.

Reply:
Will do

Reply:
You should keep all of hte Remove globs for types you're not using.  This way you don't duplicate items in the project (i.e. if the default glob would detect it as a None and the deployment glob treats it as a Content)